.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql://postgres@localhost/opencraft'
+HUEY_ALWAYS_EAGER=true
+OPENSTACK_USER='test'
+OPENSTACK_PASSWORD='pass'
+OPENSTACK_TENANT='test-tenant'
+OPENSTACK_AUTH_URL='http://example.com/auth/url'
+OPENSTACK_REGION='test-region'
```

Comment:
I found it easier and more portable to require that the OS_* variables are in the environment. The problem is that the set of variables and they way they are interpreted varies a lot between OpenStack providers. The **openstack** cli deals with this variance and because it is the standard cli, all providers make sure it just works.